README.md

```diff
@@ -3,29 +3,23 @@ Lakes
 
 
 Current features:  
--polygon lakes based on distorted ellipsoids  
+-lakes based on distorted circles
 -underground and surface lakes and lava lakes  
--lava lakes can be turned on and off  
--lakes will create a small cave above them  
--desert lakes are less probable and will turn into palm tree oases
+-underground will create a small cave above them, and surface lakes will smooth out their shorelines a little
+-desert lakes have palm trees around them
 
 Known issues:  
--Missing biome modification to prevent grass and trees to spawn on lakes  
-~~-surface lakes have difficulties appearing in perlin and other non flat world gens (unknown reason)~~  
--the "tubs" of surface lakes are sometimes incomplete  
+- Occasionally, lakes have uneven surfaces
```

Comment:
I assume it's something to do with lakes intersecting, although I don't know how that would be possible. It is at least very rare, and if you have FlowingLiquids enabled, it would quickly stabilise to a lower flat surface anyway, so I just decided to not worry about it.